### PR TITLE
Adds more content to index.html and creates 3 new html files

### PR DESCRIPTION
**Pull request by jonaner**
The 3 new html files are tennis.html, travel.html, and personalblog.html.

### CODE REVIEW

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+    <meta charset="UTF-8">
+    <title>What Makes Me Me? </title>
```
Comment (jbir789) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html:5]: nit: you can remove the extra white space

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/tennis.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> My Tennis Journey </title>
```
Comment (jbir789) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/tennis.html:5]: same here

**Comment by jbir789** [comment]
The title / desc look good!

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html (hunk 1 above)
Comment (ginellegaisano) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html:5]: I think Julien was referring to the whitespace after "Me?"

Comment (jonaner) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html:5]: Okay thank you for that clarification.

**Pull request merged**